config/__init__.py

```diff
@@ -136,6 +137,11 @@ def update_status_config(milestones_content):
 # 2015-09-14 20:50:19,185 INFO: Thing_To_Log [in /codepath/views.py:127]
 
 # set the tempdir to somewhere predictable, no matter the platform
+try:
+    os.makedirs(os.path.join(os.getcwd(), 'tmp'))
+except OSError as exception:
+    if exception.errno != errno.EEXIST:
+        raise
 tempfile.tempdir = os.path.join(os.getcwd(), 'tmp')
```

Comment:
nit:

The code computes twice the same thing with  `os.path.join(os.getcwd(), 'tmp')`
Maybe the first thing to do.

```python
tempdir_path = os.path.join(os.getcwd(), 'tmp')
```
then use it. In case we decide to change the name one day. Who knows.

Reply:
cool, will do.